queries.sql

```diff
@@ -15,3 +15,101 @@ SELECT * from animals WHERE neutered = true;
 SELECT * from animals WHERE name NOT IN ('Gabumon');
 
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+
+BEGIN;
+UPDATE animals
+SET species = 'Unspecified';
+ROLLBACK;
+SELECT species FROM animals;
```

Comment:
Great job @prg-04 🤝

 As stated by the previous reviewer, kindly verify whether the change was made or not. You never verified the changes before the rollback. Don't forget to update your results in the screenshots.
![transaction-2](https://github.com/prg-04/Vet_Clinic/assets/63917725/30b3abd9-1f9a-4d9c-a854-cb0415b3537d)